### PR TITLE
Added community is live screen

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/5647

## Description of Changes
- Added community is live screen

## "How We Fixed It"
N/A

## Test Plan
- Enable FLAG_NEW_CREATE_COMMUNITY flag
- Visit http://localhost:8080/createCommunity
- Click on the next button that has an arrow, to visit step 3 (aka success step)
- Now CA (click around) and validate UI styles

## Deployment Plan
This PR is among a series of PRs for the new create community flow.

## Other Considerations
N/A

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/SuccessStep/SuccessStep.tsx [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
 import './SuccessStep.scss';
 
 const SuccessStep = () => {
-  return <div className="SuccessStep">SuccessStep</div>;
+  return (
+    <div className="SuccessStep">
+      <img src="../../static/img/communityIslive.png" alt="" className="img" />
+      <CWText type="h2">Your community is live!</CWText>
+      <CWText type="b1">
+        Your community is now live and discoverable on Common. You can now use
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/SuccessStep/SuccessStep.tsx:13]: I think this is gray color on the designs. Yours is black

**Review by masvelio — APPROVED** [review]
one small font color change and g2g

File: packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/SuccessStep/SuccessStep.tsx (hunk 1 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/SuccessStep/SuccessStep.tsx:13]: done

**Comment by mzparacha** [comment]
Updated @masvelio

**Review by lzach83 — APPROVED** [review]
LGTM

**Pull request merged**